### DISCUSSION TITLE
Reading SNZB-02 Sensor Data from ZB-GW03 using Web Interface

### DISCUSSION THREAD

**Discussion by afgcons** [arendst/Tasmota#18617] [Q&A]
I followed the [Zigbee Tasmota](https://tasmota.github.io/docs/Zigbee/#sonoff-snzb-02-sensor) tutorial. Tasmota ! Bravo !!

But... now I am stuck - I can't find the correct command syntax to request the current or last reported sensor data (json) using a web GET request: http://tasmotazb.loc/co?xx={...}

Perhaps the "reverse" - the ZB-GW03 sends me the sensor data as it receives them. Is there a 'rules'-like option to supply the URL of my web server or application to Tasmota ?

Please, can anyone point the finger at a "How to ..." web page or provide an example of the web interface command which returns current json format sensor data,

ZB-GW03 (factory pre-flashed)
Program Version	12.3.1(tasmota)
Build Date & Time	2023-02-15T17:18:19
Core/SDK Version	2_0_5_3/v4.4.3

**Comment by sfromis** [comment]
The ZbInfo command may work for that (don't think that it queries the sensor), but for monitoring it is very much recommended to use the better MQTT strategy, where you get data as they come in, instead of http and polling. The "M" in the "Tasmota" name means "MQTT", for good reason. Anything you have in the published JSON payloads are also available to rules.
